src/components/Markdown.svelte

```diff
@@ -13,6 +13,15 @@
         start ? `start="${start}"` : ""
       } class="mzp-u-list-styled">${body}</${outerEl}>`;
     },
+    heading(text, level, raw) {
+      const id = raw
+        .toLowerCase()
+        .trim()
+        .replace(/<[!/a-z].*?>/gi, "")
```

Comment:
Idk what this is meant to remove (html tags?), but I wonder if you might have meant to put `trim()` after this (or both before and after), since if there is a string like `Some text <something>` this will return `some-text-`

Reply:
See https://github.com/mozilla/glean-dictionary/pull/2079#discussion_r1482633233
This essentially is what GitHub also does.
`some-text-` is fine. `trim()` only removes whitespace and has no effect on `some-text-`

Reply:
Moving `trim()` to after line 20 would remove the trailing space before it becomes a dash. I suppose you're right that GitHub's implementation is the same. Just did [a test](https://github.com/aminomancer/uc.css.js/blob/e1dd07a1a191d48d46b8436897a631a63a107fa5/README.md#header-test-) and it does yield a trailing dash. It's just a nitpick though. Thanks for working on this in any case!

Reply:
oh THAT space. Sorry, yes, that would be removed. I don't mind either way, this is just to get "clean" slugs to use. github-compat was just the first thing I encountered and it seemed good enough.